docs/setting-up/client/index.md

```diff
@@ -494,4 +495,4 @@ pmm-admin remove <service-type> <service-name>
 [Red Hat/CentOS/Oracle 8]: https://www.percona.com/downloads/pmm2/{{release}}/binary/redhat/8/
 [Ubuntu 16.04 (Xenial Xerus)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/xenial/
 [Ubuntu 18.04 (Bionic Beaver)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/bionic/
-[Ubuntu 20.04 (Focal Fossa)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/focal/
```

Comment:
Why do we remove Focal here?

We need to remove Xenial. + I don't see Debian 11 (bullseye) here.